app/views/layouts/application.html.erb

```diff
@@ -21,6 +20,16 @@
         <div class="it-logo"></div>
       </div>
     </header>
+    <% flash.each do |name, msg| %>
+      <% if msg.is_a?(String) %>
```

Comment:
Previous solution? I used this because it worked with foundation with no problems